zwave_js_server/model/association.py

```diff
@@ -0,0 +1,25 @@
+"""Provide a model for the association."""
+from typing import Dict, List, Optional
+from dataclasses import dataclass
+
+
+@dataclass
+class AssociationGroup:
+    """Represent a association group dict type."""
+
+    maxNodes: int  # pylint: disable=invalid-name
```

Comment:
We should use snake_case names for the attributes that we expose as interface to users of the library.